src/main/java/com/dougnoel/sentinel/webdrivers/WindowsDriverFactory.java

```diff
@@ -0,0 +1,90 @@
+package com.dougnoel.sentinel.webdrivers;
+import com.dougnoel.sentinel.system.FileManager;
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+import io.appium.java_client.service.local.AppiumServiceBuilder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import com.dougnoel.sentinel.configurations.Configuration;
+import io.appium.java_client.windows.WindowsDriver;
+import static io.appium.java_client.service.local.flags.GeneralServerFlag.BASEPATH;
+
+/**
+ * @author dougnoel
+ *
+ *         Creates WinAppDriver.
+ */
+public class WindowsDriverFactory {
+	private static final Logger log = LogManager.getLogger(WindowsDriverFactory.class);
+	private static Integer numberOfDriversRunning = 0;
+	private static final String STDOUT = "logs/WinAppDriver.log";
+	private static final String STDERR = "logs/WinAppDriverError.log";
+	private static AppiumDriverLocalService appiumService;
+
+	/**
+	 * Exists to defeat instantiation.
+	 */
+	private WindowsDriverFactory() {}
+
+	protected static void startAppiumService() {
+		AppiumServiceBuilder builder = new AppiumServiceBuilder();
+		builder.withIPAddress("127.0.0.1").withArgument(BASEPATH , "/wd/hub").usingPort(4725);
+		appiumService = AppiumDriverLocalService.buildService(builder);
+		appiumService.isRunning();
```

Comment:
Will this throw an exception if the service isn't running? If not, and it returns a boolean, should we do something with the return value?

Reply:
If something goes wrong and the service fails to start (or the isRunning check fails) it fails out and stops, which I think is the behavior we want? I'm really using isRunning like an implicit wait, since earlier on the tests would start before the Appium server was properly spun up.